CHANGELOG.md

```diff
@@ -99,6 +99,7 @@
 * [BUGFIX] Experimental TSDB: fixed memory leak in ingesters. #2586
 * [BUGFIX] Ingester: Fix an ingester starting up in the JOINING state and staying there forever. #2565
 * [BUGFIX] QueryFrontend: fixed a panic (`integer divide by zero`) in the query-frontend. The query-frontend now requires the `-querier.default-evaluation-interval` config to be set to the same value of the querier. #2603
+* [BUGFIX] Experimental TSDB: when the querier receives a `/series` request with a time range older than the data stored in the ingester, it now returns ignores the request time range and returns known series anyway instead of returning an empty response. This aligns the behaviour with the chunks storage. #2617
```

Comment:
```suggestion
* [BUGFIX] Experimental TSDB: when the querier receives a `/series` request with a time range older than the data stored in the ingester, it now ignores the requested time range and returns known series anyway instead of returning an empty response. This aligns the behaviour with the chunks storage. #2617
```